sagan-client/src/feature/clipboardButtons/main.js

```diff
@@ -8,14 +8,15 @@ module.exports = function () {
     return new Clipboard();
 };
 
-var ZeroClipboard = require('ZeroClipboard');
+var ZeroClipboard = require('zeroclipboard');
 var clipboardProvider = require('./buttonConnector');
 var copyButtonProvider = require('./copyButtonProvider');
-var buttonTemplate = require('text!./button.html');
+var buttonTemplate = '<button class="copy-button snippet"></button>';
 var $ = require('jquery');
 
-// Bootstrap is included to implicitly import the tooltip plugin. ick.
-require('bootstrap');
+// This version of bootstrap looks for window.jQuery. ick.
+window.jQuery = $;
```

Comment:
@bclozel Any idea of how to do this better? Looked into systemjs shims for several hours to no avail, figured I'd do the hackyness and spend more time seeing if I can find a better way later.